src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/AccessiblePackageIdentity.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using NuGet.Packaging.Core;
+using NuGet.Versioning;
+
+namespace NuGet.PackageManagement.UI
+{
+    /// <summary>
+    /// Represents a package identity with an acessible name in the Preview Window Dialog in the PM UI
+    /// </summary>
+    public sealed class AccessiblePackageIdentity : PackageIdentity
+    {
+        private Lazy<string> _automationNameLazy;
```

Comment:
`readonly`

Is `Lazy<T>` really important?  Does it buy us much?  Can it just be `readonly string`?

Reply:
Same as https://github.com/NuGet/NuGet.Client/pull/3217#discussion_r376162554